tensorflow_datasets/core/api_utils_test.py

```diff
@@ -20,14 +20,18 @@
 from __future__ import print_function
```

Comment:
Thank you very much for updating. I think we should only focus on the non-test files for now, as typing is not checked in tests. And this will simplify updating the project.
You can reverse changes here and the other file if this is good for you.

Reply:
Done